core/src/main/java/com/linecorp/armeria/server/ArmeriaDefaultHttpRequest.java

```diff
@@ -0,0 +1,138 @@
+package com.linecorp.armeria.server;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.linecorp.armeria.common.RequestHeaders;
+import com.linecorp.armeria.common.RequestHeadersBuilder;
+import com.linecorp.armeria.internal.common.ArmeriaHttpHeaders;
+
+import io.netty.handler.codec.http.DefaultHttpMessage;
+import io.netty.handler.codec.http.HttpHeaders;
+import io.netty.handler.codec.http.HttpMethod;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpVersion;
+import io.netty.util.internal.ObjectUtil;
+
+public class ArmeriaDefaultHttpRequest extends DefaultHttpMessage implements HttpRequest {
```

Comment:
```suggestion
final class ArmeriaDefaultHttpRequest extends DefaultHttpMessage implements HttpRequest {
```
We can remove the public modifiers in this class.